jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -260,6 +260,98 @@ describe('UserProfile', () => {
+        listModels={sinon.stub().callsArgWith(0, null, {models: null})}
+        switchModel={sinon.stub()}
+        interactiveLogin={true}
+        changeState={sinon.stub()}
+        pluralize={pluralize}
+        hideConnectingMask={sinon.stub()}
+        showConnectingMask={sinon.stub()}
+        storeUser={sinon.stub()}
+        user={users.charmstore} />, true);
+    var instance = component.getMountedInstance();
+    var output = component.getRenderOutput();
+    /* eslint-disable max-len */
+    var expected = (
+      <juju.components.Panel
+        instanceName="user-profile"
+        visible={true}>
+        <div className="twelve-col">
+          <div className="inner-wrapper">
+            <juju.components.UserProfileHeader
+              users={users}
+              avatar=""
+              interactiveLogin={instance._interactiveLogin}
+              links={links}
+              username={users.charmstore.usernameDisplay} />
+            <div className="user-profile__empty twelve-col no-margin-bottom">
+              <div className="user-profile__create-new user-profile__empty-button collapsed">
+                <form onSubmit={instance.createAndSwitch}>
+                  <juju.components.GenericButton
+                    action={instance._nextCreateStep}
+                    type='inline-neutral first'
+                    title='Create new' />
+                  <juju.components.DeploymentInput
+                    placeholder="untitled_model"
+                    required={true}
+                    ref="modelName"
+                    validate={[{
+                      regex: /\S+/,
+                      error: 'This field is required.'
+                    }, {
+                      regex: /^[a-zA-Z0-9]([a-zA-Z0-9.-]*[a-zA-Z0-9])?$/,
+                      error: 'This field must only contain upper and lowercase ' +
+                        'letters, numbers, and hyphens. It must not start or ' +
+                        'end with a hyphen.'
+                    }]} />
+                  <juju.components.GenericButton
+                    action={instance.createAndSwitch}
+                    type='inline-neutral second'
+                    title='Submit' />
+                </form>
+              </div>
+              <div className="clearfix">
+                <img alt="Empty profile"
+                  className="user-profile__empty-image"
+                  src="/static/gui/build/app/assets/images/non-sprites/empty_profile.png" />
+                <h2 className="user-profile__empty-title">
+                  We are unable to load your profile at the moment
+                </h2>
+                <p className="user-profile__empty-text">
+                  Your models, bundles and charms will appear here when the problem is resolved.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </juju.components.Panel>
+    );
+    /* eslint-disable max-len */
```

Comment:
OK, now that I've investigated more... it's there because quite a bit of the JSX in creating the expected response runs over the 80 char limit. The same thing is done in the existing "renders empty state" test.